app/build.gradle.kts

```diff
@@ -0,0 +1,59 @@
+android {
```

Comment:
Let's add a big comment on this file saying something like "hey we R cool kidz so we use the Kotlin DSL.  If you're writing an app that uses FirebaseUI check out README.md for better examples" or something like that (please not that verbatim)

Reply:
Haha, I'm definitely going to quote you on that one! 😁 Side note: do you think we should rename `:app` to `:sample`?